src/tools/plotter/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(iDynTreePlotter)
+
+find_package(Qt5 COMPONENTS Widgets Charts)
+if (${Qt5_FOUND})
```

Comment:
Sorry, I meant `OPTIONAL_COMPONENTS`, as in https://cmake.org/cmake/help/v3.0/command/find_package.html .